.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+install: true
+script:
+  - '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
+  - '[ "${TRAVIS_PULL_REQUEST}"  = "false" ] || ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
```

Comment:
yeah, I copied this from SCDF where we had different flow PR and main commit. hence these lines. will remove on of them.